tests/spread/general/environment/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: container environment test
+
+execute: |
+  rockcraft pack
+
+  test -f environment-test_latest.rock
+  test ! -d work
+
+  # test container execution
+  docker images
+  sudo /snap/rockcraft/current/bin/skopeo --insecure-policy copy oci-archive:environment-test_latest.rock docker-daemon:environment-test:latest
+  docker images
+  docker run environment-test bash -c 'echo $X' | grep "ship it!"
```

Comment:
if cmd is set to /bin/bash why add the "bash"?

Reply:
If a command is defined in the command line it replaces what's defined as the container `cmd`.

Reply:
I'll change the default cmd in the test to make the test more meaningful.